io.github.am2rcommunitydevelopers.AM2RLauncher.yaml

```diff
@@ -0,0 +1,134 @@
+app-id: io.github.am2rcommunitydevelopers.AM2RLauncher
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.gnome.Sdk.Compat.i386
+
+finish-args:
+# TODO: crashes with wayland on filepicker and font being whacky. Likely Eto bug, test and report there.
+- --socket=x11
+- --share=ipc
+
+# Network needed for updates, pulseaudio for sound
+- --share=network
+- --socket=pulseaudio
+
+# Home for getting the 1.1 zip, config because of profiles having their saves there, apktool for apk creation
+- --filesystem=home:ro
+- --filesystem=~/.config/
+- --filesystem=xdg-data/apktool/:create
+
+# libappindicator stuff
+- --talk-name=com.canonical.indicator.application
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+
+# Fuse / Appimage related
+- --talk-name=org.freedesktop.Flatpak.*
+- --device=all
+- --filesystem=/tmp
+
+# Java and x32 stuff
+- --env=JAVA_HOME=/app/jre
+- --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+- --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
+
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '42'
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: '21.08'
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+command: AM2RLauncher
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: libappindicator
+
+  - name: xdelta3
+    sources:
+      - type: archive
+        url: https://github.com/jmacd/xdelta/archive/refs/tags/v3.1.0.tar.gz
+        sha256: 7515cf5378fca287a57f4e2fee1094aabc79569cfe60d91e06021a8fd7bae29d
+    buildsystem: simple
+    build-commands:
+      - mv ./xdelta3/* ./
+      - rm -r ./xdelta3
+      - autoreconf --install
+      - ./configure
+      - make
+      - mv ./xdelta3 /app/bin/xdelta3
```

Comment:
how about using `buildsystem: autotools` and using `subdir: xdelta3`. It even has an install target. And this will configure the build properly which you don't do here.

Reply:
Hmm, this doesn't seem to work on my end
Using
```yaml
  - name: xdelta3
    sources:
      - type: archive
        url: https://github.com/jmacd/xdelta/archive/refs/tags/v3.1.0.tar.gz
        sha256: 7515cf5378fca287a57f4e2fee1094aabc79569cfe60d91e06021a8fd7bae29d
    buildsystem: autotools
    subdir: xdelta3
```
flatpak-builder does recognize the subdir parameter, but doesn't seem to actually navigate inside it, as I'm getting the "can't find autogen / bootstrap" error.

Reply:
You can add the followingn source:
```yaml
- type: shell
  commands:
    - autoreconf -si
```

Reply:
That helped, thanks!